### DISCUSSION TITLE
3.3rc0

### DISCUSSION THREAD

**Discussion by snowman2** [pyproj4/pyproj#998] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/pyproj4/pyproj/releases/tag/3.3rc0'>3.3rc0</a>.</em>

**Comment by snowman2** [comment]
The plan is to release with PROJ 8.2 in the wheels.

Release Candidate 0 Status:

- [x] Linux/OSX wheels (https://github.com/pyproj4/pyproj-wheels/pull/44)
- [ ] Windows wheels (@cgohlke)
- [x] Conda Forge (https://github.com/conda-forge/pyproj-feedstock/pull/113)
- [x] Debian (@sebastic)
- [x] Fedora (@jdkloe)

Plan to release sometime next weekend. Testing is always appreciated, but definitely not required.

**Comment by sebastic** [comment]
The Debian package builds are looking good so far:

 https://buildd.debian.org/status/package.php?p=python-pyproj&suite=experimental

**Reply by snowman2** [reply]
Thanks @sebastic 👍

**Comment by jdkloe** [comment]
Building and testing on fedora rawhide runs just fine.
The only thing that could be improved is the fact that I get some warnings:
* a deprecation warning that should be fixed in an upcoming version.
* some no-network warnings. These are expected, since I test in a mock environment without network. But I don't think it is correct to issue warnings in this case, since there is nothing wrong with installing in a no-network environment.

```
==================================== warnings summary ====================================
../../usr/lib64/python3.10/site-packages/numexpr/expressions.py:20
  /usr/lib64/python3.10/site-packages/numexpr/expressions.py:20: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
    from distutils.version import LooseVersion

test/test_datadir.py::test_get_data_dir__from_user[str]
test/test_datadir.py::test_get_data_dir__from_user[Path]
test/test_datadir.py::test_append_data_dir__internal[str]
test/test_datadir.py::test_append_data_dir__internal[Path]
  /manualbuilddir/pyproj-3.3rc0/build/lib.linux-x86_64-3.10/pyproj/datadir.py:38: UserWarning: pyproj unable to set database path.
    _global_context_set_data_dir()
test/test_transformer.py::test_transformer_group__unavailable
test/test_transformer.py::test_transformer_group__network_disabled
test/test_transformer.py::test_transformer_group__download_grids__directory
  /manualbuilddir/pyproj-3.3rc0/build/lib.linux-x86_64-3.10/pyproj/transformer.py:184: UserWarning: Best transformation is not available due to missing Grid(short_name=us_noaa_alaska.tif, full_name=, package_name=, url=https://cdn.proj.org/us_noaa_alaska.tif, direct_download=True, open_license=True, available=False)
    super().__init__(

test/test_transformer.py::test_transformer_group__get_transform_crs
  /manualbuilddir/pyproj-3.3rc0/build/lib.linux-x86_64-3.10/pyproj/transformer.py:184: UserWarning: Best transformation is not available due to missing Grid(short_name=nl_nsgi_rdtrans2018.tif, full_name=, package_name=, url=https://cdn.proj.org/nl_nsgi_rdtrans2018.tif, direct_download=True, open_license=True, available=False)
    super().__init__(

-- Docs: https://docs.pytest.org/en/stable/warnings.html
=============== 799 passed, 1 skipped, 24 deselected, 9 warnings in 22.82s ===============
```

I tested using these versions:
python 3.10.0
gcc 11.2.1
proj 8.2.0

**Comment by snowman2** [comment]
Thanks @jdkloe 👍

- The depreciation looks like it is coming from an external library, so I think we're okay. If I missed something, please let me know.
- The warnings from the TransformerGroup look like they are behaving properly since the grid is missing. The tests probably need to be tweaked to catch the warning in that scenario.

**Comment by jdkloe** [comment]
The deprecation indeed comes from the numexpr package, which seems to be popping up since it is a dependency of pandas which is used during the test run. De issue seems to be solved already in the numexpr development version, see:
https://github.com/pydata/numexpr/blob/master/numexpr/expressions.py
and the last comment in https://github.com/pydata/numexpr/issues/312
so there is no action needed on pyproj side.

Anyway, these are all warnings, so no reason at all to block the release that you proposed.

**Reply by snowman2** [reply]
Sounds good, thanks for looking into it :+1:

**Comment by snowman2** [comment]
I got a free moment, so it is here a couple of days early: https://github.com/pyproj4/pyproj/releases/tag/3.3.0

Thanks again all!

**Comment by cgohlke** [comment]
Wheels for Windows are at <https://www.lfd.uci.edu/~gohlke/pythonlibs/#pyproj>.

**Reply by snowman2** [reply]
Thanks @cgohlke  👍
